tests/nnfw_api/src/one_op_tests/Conv2D.cc

```diff
@@ -35,7 +35,7 @@ TEST_F(GenModelTest, OneOp_Conv2D)
   _context->addTestCase(uniformTCD<float>(
       {{4, 0, -5, 1, 0, 4, -1, 1, -1, -3, 3, -2, -4, 1, -2, 2, 4, -4, 2, 2, 0, 4, -1, -2, 4}},
       {{47, -4, -25, 9, 10, 10, -13, 11, -14, -26, -12, 26, 20, 40, 1, 3, 11, 4}}));
-  _context->setBackends({"acl_cl", "acl_neon", "cpu", "ruy"});
+  _context->setBackends({"acl_cl", "acl_neon", "cpu", "ruy", "xnnpack"});
```

Comment:
I just found that you have to add `xnnpack` to `GenModelTestContext::setBackends`. https://github.com/Samsung/ONE/blob/c46ddc04abdb58323fbd38389e6927f003bfaea1/tests/nnfw_api/src/GenModelTest.h#L215

Without that line, `xnnpack` will not be tested on `GenModelTest`.